it-and-security/teams/workstations.yml

```diff
@@ -0,0 +1,60 @@
+name: "Workstations"
+team_settings:
+  features:
+    enable_host_users: true
+    enable_software_inventory: true
+  host_expiry_settings:
+    host_expiry_enabled: false
+    host_expiry_window: 0
+  secrets:
+    - secret: $DOGFOOD_WORKSTATIONS_ENROLL_SECRET
+agent_options:
+  path: ../lib/agent-options.yml
+controls:
+  enable_disk_encryption: true
+  macos_settings:
+    custom_settings:
+      - path: ../../mdm_profiles/automatic_updates.mobileconfig
```

Comment:
Ah, I see we're referencing the `mdm_profiles/`

I think we want to move this under the `lib/` folder so we're following the best practice. Maybe we do that in a separate PR after this gets merged?

Also, heads up that there's some GitHub workflows cleanup that we want to do as well (from [the user story](https://github.com/fleetdm/fleet/issues/17043)):
 
![Screenshot 2024-02-27 at 5 24 50 PM](https://github.com/fleetdm/fleet/assets/47070608/96d8a65c-29d1-409f-93d9-4fb7fba729a0)

Reply:
Yes, let's move `mdm_profiles` in another PR. I deleted the old workflows.